shell/platform/windows/windows_registry.cc

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/windows_registry.h"
+
+namespace flutter {
+
+WindowsRegistry::WindowsRegistry() {}
```

Comment:
You can delete this if you apply the default suggestion below. Sadly I'm not GitHub-suggestion-savvy enough to know how to delete two lines in a suggestion :/